speaker/layer2_controller_test.go

```diff
@@ -22,7 +22,6 @@ func (sl *fakeSpeakerList) UsableSpeakers() map[string]bool {
 }
 
 func (sl *fakeSpeakerList) Rejoin() {
-	return
```

Comment:
I would collapse the braces if the function block is empty.

Reply:
Done